apsuite/commissioning_scripts/meas_coupling_tune.py

```diff
@@ -1,14 +1,38 @@
 """Coupling Measurement from Minimal Tune Separation."""
 
+import time as _time
+from threading import Thread as _Thread, Event as _Event
 import numpy as _np
+from scipy.optimize import least_squares
 import matplotlib.pyplot as _plt
 
+from siriuspy.devices import PowerSupply, Tune
+from .base import BaseClass
 
-from ..optimization import SimulAnneal as _SimulAnneal
 
+class CouplingParams():
+    """."""
+
+    def __init__(self):
+        """."""
+        self.nr_points = 1
```

Comment:
I think the default value of this attribute should be more appropriate, @murilobalves.